### PR TITLE
[REF] Minor layout adjustments

### LINKED ISSUE #61: Minor layout improvements
- [x] Less margins around map / make parent container larger
- [x] Use 0% map colormap padding for more state differentiation
- [x] Turn off map scrollZoom (makes scrolling down page harder since map area is so big)
- [x] Change colorbar titles (?)
  - Percentage (impact) -> percentage (weather event)
  - Percentage (opinion) -> percentage (endorsement)

### LINKED ISSUE #63: Deselect state if same state clicked twice in map

### PR DESCRIPTION
**Pull request by alyssadai**
- Closes #61
- Closes #63 
- Also removes some leftover `TODO`s

### CODE REVIEW

File: climate_emotions_map/make_map.py
Comment (alyssadai) [inline climate_emotions_map/make_map.py:1]: Hey @michellewang, just keeping you updated with some minor formatting changes I'm making in your `make_map.py`. :)

No need to review the whole PR, but let me know if any changes I made to your code look problematic (seems okay based on some local testing)!

File: climate_emotions_map/utility.py [hunk 1]
```diff
@@ -25,7 +25,6 @@
 
 def get_state_options():
     """Get the options for the state dropdown."""
-    # TODO: The state values include cluster labels in parentheses, e.g. (Cluster F). Can remove from label if desired.
```
Comment (michellewang) [inline climate_emotions_map/utility.py:28]: This would be an app-wide change, right? Since the state dropdown shows the cluster labels too

Comment (michellewang) [reply climate_emotions_map/utility.py:28]: Oh just realized that this is not a change in `make_map.py` 😅 but my question still holds I think -- if the "(Cluster X)" is stripped then it needs to be stripped from the map hoverboxes too

File: climate_emotions_map/make_map.py
Comment (michellewang) [reply climate_emotions_map/make_map.py:1]: Without actually running it I agree that it should be okay!

File: climate_emotions_map/utility.py (hunk 1 above)
Comment (alyssadai) [reply climate_emotions_map/utility.py:28]: It would, but actually Eric already confirmed we don’t need to change this, so we’re good!!

**Pull request merged**